ControlRoomApplication/ControlRoomApplication/Controllers/RadioTelescopeControllers/RadioTelescopeController.cs

```diff
@@ -791,6 +798,69 @@ private void SensorMonitor()
                 azTempSafe = checkTemp(azTemp, azTempSafe);
                 elTempSafe = checkTemp(elTemp, elTempSafe);
 
+                // Sensor status routine, checks for each sensor to update the status in the DB
+                // Check Gate
+                SensorStatusEnum gate = SensorStatusEnum.NORMAL;
+
+                // Check azimuth temp 1
+                SensorStatusEnum azTemp1 = SensorStatusEnum.NORMAL;
+
+                // Check azimuth temp 2
+                SensorStatusEnum azTemp2 = SensorStatusEnum.NORMAL;
+
+                // Check elevation temp 1
+                SensorStatusEnum elTemp1 = SensorStatusEnum.NORMAL;
+
+                // Check elevation temp 2
+                SensorStatusEnum elTemp2 = SensorStatusEnum.NORMAL;
+
+                // Check weather
+                SensorStatusEnum weather = SensorStatusEnum.NORMAL;
+
+                // Check elevation absolute encoder, set to ALERT if timed out
+                SensorStatusEnum elAbsEncoder = SensorStatusEnum.NORMAL;
+                if (RadioTelescope.PLCDriver.MotorsCurrentlyMoving())
+                {
+                    if (prevElevation == GetCurrentOrientation().Elevation)
```

Comment:
Just a reminder, but the prevElevation might always be changing depending on how we fix the noise issue with the absolute encoder. This is fine for now, but I just want to mention that it may need to be changed depending on how the issue is fixed.